docs/guide.md

```diff
@@ -478,9 +478,9 @@ scala> Stream(1,2,3).merge(Stream.eval(IO { Thread.sleep(200); 4 })).runLog.unsa
 res38: Vector[Int] = Vector(1, 2, 3, 4)
 ```
 
-The `merge` function is defined in [`pipe2`](../core/shared/src/main/scala/fs2/pipe2.scala), along with other useful concurrency functions, like `interrupt` (halts if the left branch produces `false`), `either` (like `merge` but returns an `Either`), `mergeHaltBoth` (halts if either branch halts), and others.
+The `merge` function supports concurrency. FS2 has a number of other useful concurrency functions like `concurrently` (runs another stream concurrently and discards its output), `interrupt` (halts if the left branch produces `false`), `either` (like `merge` but returns an `Either`), `mergeHaltBoth` (halts if either branch halts), and others.
```

Comment:
I think in 0.9 this would have completely different behaviour than in 0.10

```scala

val s1: Stream[Int] = ???

val s2: Stream[Boolean]  = time.sleep_(100 millis) ++ Stream.emit(true)

(s1 interruptWhen s2) flatMap { i =>
 Stream.emit(i) ++ sleep_(1.minute)
}
```
In 0.9 this will interrupt s1 in 100 millis, however in 0.10 this will interrupt s1 in 1.minute